frontend/src/game/hand.js

```diff
@@ -23,8 +23,9 @@ module.exports = {
 
         handState.forEach(function (cardInHand, index) {
             var newCard = templates.baseCard.cloneNode(true);
-            newCard.innerHTML = cardInHand["power"]
-            newCard.innerHTML += showAddPower(cardInHand["add-power"])
+            newCard.innerHTML = cardInHand["power"];
+            newCard.setAttribute("add-power", cardInHand["add-power"]);
+            newCard.innerHTML += showAddPower(cardInHand["add-power"]);
```

Comment:
The ability `add-power` is set as an attribute, so we can easily check it for a card.
Maybe this two lines can be simplified.